.github/workflows/stage-lint.yml

```diff
@@ -15,6 +15,9 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v2
+      # Fixes `error obtaining VCS status: exit status 128`
+      # https://github.com/golangci/golangci-lint/issues/4033
+      - run: git config --global --add safe.directory $GITHUB_WORKSPACE
```

Comment:
Would upgrading to `actions/checkout@v4` avoid the need to do this? It seems like `actions/checkout` has the capability to do this automatically by default.

Reply:
I upgraded to actions/checkout@v4 and it did not alleviate the error I was facing. I tried adding `set-safe-directory: true` on top of it and that did not work either. I'm going to leave it as it is.